G1ANT.Addon.Net/G1ANT.Addon.Net/Commands/MailClearAttachmentsCommand.cs

```diff
@@ -0,0 +1,36 @@
+﻿/**
+*    Copyright(C) G1ANT Ltd, All rights reserved
+*    Solution G1ANT.Addon, Project G1ANT.Addon.Net
+*    www.g1ant.com
+*
+*    Licensed under the G1ANT license.
+*    See License.txt file in the project root for full license information.
+*
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using MailKit;
+using MailKit.Net.Imap;
+using G1ANT.Language;
+using G1ANT.Language.Services;
+using System.Net;
+
+
+namespace G1ANT.Addon.Net
+{
+    [Command(Name = "mail.clearattachments", Tooltip = "This command tries to delete all previouslz downloaded attachments.")]
```

Comment:
previouslZ -> previouslY

Reply:
Does the command 'try'? means there's a possibility for it to fail? If so, there's some kind of exception thrown / a message shown?

Reply:
No, if there are no attachments noething will be deleted.